crates/libcontainer/src/syscall/linux.rs

```diff
@@ -38,87 +38,65 @@ const MOUNT_ATTR_NODIRATIME: u64 = 0x00000080;
 const MOUNT_ATTR_NOSYMFOLLOW: u64 = 0x00200000;
 
 /// Constants used by mount_setattr(2).
-pub enum MountAttrOption {
+pub enum MountRecursive {
```

Comment:
I like the changes to drop the `MountAttr` prefix to the enum fields. I am not sure about `MountAttrOption` -> `MountRecursive`. These are defined in the syscalls interface as `MOUNT_ATTR`, so should we keep the name `MountAttrOption` to avoid confusion?

Reply:
@utam0k Do you remember why we did not implement the rest of the mount attr options in this `enum` and choose to match string directly here? https://github.com/containers/youki/blob/main/crates/libcontainer/src/rootfs/utils.rs#L131

Reply:
Nvm, after reading through the code again, I know understand. You are correct. These options only corresponds to recursive mount options and gets passed through to `rec_attr` in `MountAttr` and then requires the set of `AT_RECURSIVE` flag when applied. I agree the `MountAttrOption` is confusing and rename to something having recursive in the name is the right thing to do.